packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -1839,7 +1976,10 @@ abstract class TreeDiagnosticsMixin implements TreeDiagnostics {
   /// common [DiagnosticsProperty] parameters.
   ///
   /// ```dart
-  /// abstract class ExampleObject extends Object with TreeDiagnosticsMixin {
+  /// abstract class ExampleObject extends ExampleSuperclass with TreeDiagnosticsMixin {
+  ///
+  ///   // ...various members and properties...
+  ///
   ///   @override
   ///   void debugFillProperties(List<DiagnosticsNode> description) {
   ///     // Always add properties from the base class first.
```

Comment:
In the dartdocs it'll look like the "// ...various members and properties..." comment is hiding the properties being described here. You won't see that ExampleSuperclass actually defines them.